relay-metrics/src/aggregator.rs

```diff
@@ -472,6 +737,21 @@ pub struct Aggregator {
 }
 
 impl Aggregator {
+    /// Create a new aggregator.
+    pub fn new(config: AggregatorConfig) -> Self {
```

Comment:
This is only used in tests, right?

```suggestion
    #[cfg(test)]
    pub fn new(config: AggregatorConfig) -> Self {
```

Reply:
no, it's used for the default aggregator in the router